path_config_test.go

```diff
@@ -204,19 +224,23 @@ func TestConfigDelete(t *testing.T) {
 
 	// Test valid config
 	config := map[string]interface{}{
-		"subscription_id":         "a228ceec-bf1a-4411-9f95-39678d8cdb34",
-		"tenant_id":               "7ac36e27-80fc-4209-a453-e8ad83dc18c2",
-		"client_id":               "testClientId",
-		"client_secret":           "testClientSecret",
-		"environment":             "AZURECHINACLOUD",
-		"root_password_ttl":       int((24 * time.Hour).Seconds()),
-		"identity_token_audience": "",
-		"identity_token_ttl":      int64(0),
+		"subscription_id":            "a228ceec-bf1a-4411-9f95-39678d8cdb34",
```

Comment:
nit: might be worth adding the CE error path test case for Azure similar to [AWS](https://github.com/hashicorp/vault/blob/main/builtin/logical/aws/path_config_root_test.go#L269)

Reply:
added